### PR TITLE
[ENH] Add line breaks in _episodes/03-lists

### PR DESCRIPTION
**Pull request by emdupre**
Helps to address #498, following up on #565 !

Changes proposed:
- Breaks up long lines (i.e., > 100 characters) in _episodes/03-lists.md

### CODE REVIEW

File: _episodes/03-lists.md [hunk 1]
```diff
@@ -111,17 +111,19 @@ does not.
...
+> Lists and arrays, on the other hand, are mutable: we can modify them after they have been created.
+> We can change individual elements, append new elements, or reorder the whole list.
+> For some operations, like sorting, we can choose whether to use a function that modifies the data
+> in-place or a function that returns a modified copy and leaves the original unchanged.
 >
-> Be careful when modifying data in place.  If two variables refer to the same list, and you modify
-> the list value, it will change for both variables!
+> Be careful when modifying data in-place.
+> If two variables refer to the same list, and you modify the list value,
+> it will change for both variables!
```
Comment (maxim-belkin) [inline _episodes/03-lists.md]: Let's keep all the lines above unchanged. They do not exceed the 100-character limit.

File: _episodes/03-lists.md [hunk 2]
```diff
@@ -173,7 +175,8 @@ does not.
 > Here is a visual example of how indexing a list of lists `x` works:
 >
 > <a href='https://twitter.com/hadleywickham/status/643381054758363136'>
-> ![The first element of a list. Adapted from @hadleywickham's tweet about R lists.](../fig/indexing_lists_python.png)</a>
+> ![The first element of a list. Adapted from @hadleywickham.](../fig/indexing_lists_python.png)
+> </a>
```
Comment (maxim-belkin) [inline _episodes/03-lists.md]: Let's rewrite this link in Markdown.

```
[![The first element of a list.
   Adapted from @hadleywickham.](../fig/indexing_lists_python.png)][hadleywickham-tweet]
```

And add the following to the bottom of the document:

```
[hadleywickham-tweet]: https://twitter.com/hadleywickham/status/643381054758363136
```

File: _episodes/03-lists.md [hunk 3]
```diff
@@ -214,7 +217,7 @@ does not.
 
 > ## Heterogeneous Lists
 > Lists in Python can contain elements of different types. Example:
-> ~~~ 
+> ~~~
```
Comment (maxim-belkin) [inline _episodes/03-lists.md]: 👍

File: _episodes/03-lists.md [hunk 4]
```diff
@@ -539,4 +539,6 @@ Omitting ending index: ["sep", "oct", "nov", "dec"]
 > {: .solution}
 {: .challenge}
 
+[hadleywickham-tweet]: https://twitter.com/hadleywickham/status/643381054758363136
+
 {% include links.md %}
```
Comment (maxim-belkin) [inline _episodes/03-lists.md]: Thanks! Was about to add a comment about it :)

File: _episodes/03-lists.md [hunk 5]
```diff
@@ -172,8 +172,8 @@ does not.
 >
 > Here is a visual example of how indexing a list of lists `x` works:
 >
-> <a href='https://twitter.com/hadleywickham/status/643381054758363136'>
-> ![The first element of a list. Adapted from @hadleywickham's tweet about R lists.](../fig/indexing_lists_python.png)</a>
+> [![The first element of a list.
+   Adapted from @hadleywickham.](../fig/indexing_lists_python.png)][hadleywickham-tweet]
```
Comment (maxim-belkin) [inline _episodes/03-lists.md]: I know this does not change much, but let's add `> ` in the beginning of this line.

File: _episodes/03-lists.md [hunk 6]
```diff
@@ -539,4 +539,6 @@ Omitting ending index: ["sep", "oct", "nov", "dec"]
 > {: .solution}
 {: .challenge}
 
+[hadleywickham-tweet]: https://twitter.com/hadleywickham/status/643381054758363136
```
Comment (maxim-belkin) [inline _episodes/03-lists.md]: Let's use this link on line 211 as well.

Comment (emdupre) [reply _episodes/03-lists.md]: Great catch !

**Comment by emdupre** [comment]
Thank _you_ !

**Pull request merged**